include/odp/api/spec/traffic_mngr.h

```diff
@@ -1783,6 +1789,21 @@ int odp_tm_queue_disconnect(odp_tm_queue_t tm_queue);
  */
 int odp_tm_enq(odp_tm_queue_t tm_queue, odp_packet_t pkt);
 
+/** The odp_tm_enq_multi() function is used to add packets to a given TM system.
+ * This API takes multiple packets to enqueue unlike odp_tm_enq() but
+ * rest of the behaviour is identical to that of odp_tm_enq().
+ * WRED drops donot cause failure of the call.
```

Comment:
I would like to propose this wording:

The odp_tm_enq_multi() function is used to add packets to a given TM system. This function enqueues multiple packets but is otherwise similar to odp_tm_enq(). Packets dropped by WRED or other queue management action do not cause this function to return a failure. Such packets get consumed just like the packets that are not dropped.